docs/reference/supported-environments.md

```diff
@@ -78,17 +78,17 @@ You can also use newer versions of Desktop and Web Modeler with older Zeebe vers
-| Optimize 3.7.x           | Camunda Platform 7.14.x, 7.15.x, 7.16.x         | OpenJDK 11+               | 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+ |
-| Optimize 3.8.x           | Camunda Platform 7.15.x, 7.16.x, 7.17.x         | OpenJDK 11+               | 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+        |
-| Optimize 3.9.x           | Camunda Platform 7.16.x, 7.17.x, 7.18.x         | OpenJDK 11+               | 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+                                   |
-| Optimize 3.10.x          | Camunda Platform 7.17.x, 7.18.x, 7.19.x         | OpenJDK 17+               | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+                                              |
-| Optimize 3.11.x / 7.20.X | Camunda Platform 7.17.x, 7.18.x, 7.19.x, 7.20.x | OpenJDK 17+               | 8.7.0+, 8.8.0+                                                                |
+| Improve                 | Automate                                        | Java version              | Elasticsearch version                                                         |
+| ----------------------- | ----------------------------------------------- | ------------------------- | ----------------------------------------------------------------------------- |
+| Optimize 3.3.x          | Camunda Platform 7.12.11+, 7.13.5+, 7.14.x      | OpenJDK 8+ or OpenJDK 11+ | 7.3.0+, 7.4.0+, 7.5.0+, 7.6.0+, 7.7.0+, 7.8.0+, 7.9.0+, 7.10.0+               |
+| Optimize 3.4.x          | Camunda Platform 7.13.5+, 7.14.x, 7.15.x        | OpenJDK 8+ or OpenJDK 11+ | 7.5.1+, 7.6.0+, 7.7.0+, 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+                      |
+| Optimize 3.5.x          | Camunda Platform 7.13.5+, 7.14.x, 7.15.x        | OpenJDK 11+               | 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+                            |
+| Optimize 3.6.x          | Camunda Platform 7.14.x, 7.15.x, 7.16.x         | OpenJDK 11+               | 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+          |
+| Optimize 3.7.x          | Camunda Platform 7.14.x, 7.15.x, 7.16.x         | OpenJDK 11+               | 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+ |
+| Optimize 3.8.x          | Camunda Platform 7.15.x, 7.16.x, 7.17.x         | OpenJDK 11+               | 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+        |
+| Optimize 3.9.x          | Camunda Platform 7.16.x, 7.17.x, 7.18.x         | OpenJDK 11+               | 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+                                   |
+| Optimize 3.10.x         | Camunda Platform 7.17.x, 7.18.x, 7.19.x         | OpenJDK 17+               | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+                                              |
+| Optimize 8.3.x / 3.11.x | Camunda Platform 7.17.x, 7.18.x, 7.19.x, 7.20.x | OpenJDK 17+               | 8.7.0+, 8.8.0+                                                                |
```

Comment:
Ooh I just noticed that the Engine support listed is wrong. Would you mind removing 7.17.x support for 3.11 please?